### ISSUE TITLE
Notification not working if Telegram is not open

### ISSUE THREAD

**Issue by kellykim5** [telegramdesktop/tdesktop#7424]
### Steps to reproduce
1. Have "native notification" on
2. Close Telegram (not minimized)

### Expected behaviour
Should give me notifications even when Telegram is not open.

### Actual behaviour
If Telegram is open but minimized, it gives me notifications. But if it's closed, it doesn't give me notifications nor does it show the number of messages on my desktop icon.

### Configuration
**Operating system:** Elementary OS 5.1.2 Hera 
                                    Built on Ubuntu 18.04.3 LTS
                                    Linux 4.15.0-91-generic
                                    GTK 3.22.30

**Version of Telegram Desktop:** 1.9.21

**Used theme**: Classic dark mode

<details><summary><b>Logs</b>:</summary>
Insert log.txt here (if necessary)
</details>

**Comment by ilya-fedin** [comment]
> Should give me notifications even when Telegram is not open.

How should this work? :thinking:

**Comment by kellykim5** [comment]
Let me rephrase the issue.
If I have it open (or minimized), it gives me notifications. If I just close the window, it doesn't give me notifications. It used to but after upgrading to the newest version it's not doing that any more.
For example, my mail client gives me notifications even when I don't have the mail app open - but Telegram doesn't.

**Comment by kellykim5** [comment]
> > Should give me notifications even when Telegram is not open.
> 
> How should this work? thinking

Let me rephrase the issue.
If I have it open (or minimized), it gives me notifications. If I just close the window, it doesn't give me notifications. It used to but after upgrading to the newest version it's not doing that any more.
For example, my mail client gives me notifications even when I don't have the mail app open - but Telegram doesn't.

**Comment by ilya-fedin** [comment]
https://github.com/telegramdesktop/tdesktop/issues/7252#issuecomment-597681038
https://github.com/telegramdesktop/tdesktop/issues/7252#issuecomment-597684232

**Comment by xvitaly** [comment]
Enable tray icon in settings (in Gnome 3 you must install [AppIndicator Support](https://extensions.gnome.org/extension/615/appindicator-support/) extension) and Telegram will be minimized to tray instead of closing when you press X button.

**Comment by kellykim5** [comment]
> Enable tray icon in settings (in Gnome 3 you must install [AppIndicator Support](https://extensions.gnome.org/extension/615/appindicator-support/) extension) and Telegram will be minimized to tray instead of closing when you press X button.

thanks!!
